src/cli/Cli.ts

```diff
@@ -363,6 +325,75 @@ export class Cli {
     }
   }
 
+  private async validateOrPromptForOptions(index: number, optionsLength: number, shouldPrompt: boolean, commandToExecute: CommandInfo, optionsFromArgs: any): Promise<void> {
+    if (typeof commandToExecute.options[index] !== 'undefined' &&
+      commandToExecute.options[index].required &&
+      typeof optionsFromArgs.options[commandToExecute.options[index].name] === 'undefined') {
+
+      if (!shouldPrompt) {
+        return this.closeWithError(`Required option ${commandToExecute.options[index].name} not specified`, optionsFromArgs, true);
+      }
+      if (index === 0) {
+        Cli.log("Supply values for the following parameters:");
```

Comment:
Since this isn't primary command output it should be logged to stderr so that it's not a part of command's output which could work with in a script or store in a variable

Reply:
TBH this comment is not quite clear for me 🤔. What else may I use here instead of Cli.Log() 🤔?

Reply:
I see what you mean. My first concern was to separate the prompts from the command output so that when you assign CLI's output to a variable, you'll only get the command output and not the prompts. That requires however access to logger which we don't have defined at this stage. I'm not sure if it's an issue though as we're using CLI interactively. We could use `Cli.error` which doesn't sound that intuitive but which sends output to stderr to err on the safe side. What do you think?

Reply:
well log() uses console.log(), error() uses console.error()... maybe we could also add Cli.warn() which will use console.warn() 🤔? This message seems more like warning than an error 😉 and I think would still work. Also if we would save it to some parameter we would not get the warn message in the output. Unfortunately we will get the questions in the param from inquirer prompt as well
Please see this test result were I already moved the "Provide values for the following parameters:" to console.warn output
![image](https://user-images.githubusercontent.com/58668583/154767597-93d0982a-fa8e-4467-a55a-96913888c3ec.png)

when saving to variable we don't get the message in it but we get the parameters from prompt. I am not sure will this satisfy your need?

Reply:
> maybe we could also add Cli.warn() which will use console.warn()

Since there's just stdout and stderr, what's the benefit of adding CLI.warn() over CLI.error()?

If there's no way to redirect inquirer's output to stderr, then I guess it makes the decision for us. In that case, let's keep using `Cli.log()` for now and we can always revisit it in the future. Thank you for looking into it.